geographic_msgs/CMakeLists.txt

```diff
@@ -40,6 +39,38 @@ rosidl_generate_interfaces(${PROJECT_NAME}
   DEPENDENCIES unique_identifier_msgs geometry_msgs std_msgs builtin_interfaces
 )
 
+rosidl_get_typesupport_target(cpp_typesupport_target "${PROJECT_NAME}" rosidl_typesupport_cpp)
+
+if(TARGET "${cpp_typesupport_target}")
```

Comment:
When would this not be true?

Reply:
Looking at the related PR, it appears this happens if you decide to build without the cpp generators enabled, say you are operating on a middlware that doesn't support C++. 

I didn't think this was a common thing, but if it was contributed by Pablo from eProsima to common_interfaces, they must have had a good reason. The use case was not shared in the original PR.

Reply:
OK, that's fine with me,

Reply:
I contributed the same modification using `TARGET` upstream to `sensor_msgs`: https://github.com/ros2/common_interfaces/pull/238